boards/arm/stm32_min_dev/support/openocd.cfg

```diff
@@ -0,0 +1,17 @@
+source [find interface/stlink-v2.cfg]
```

Comment:
Is that actually working with stlink?

Can you type "make BOARD=stm32_min_dev debug"
and start an openocd session on the board?